airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/json_operations.sql

```diff
@@ -68,15 +68,27 @@
 {%- endmacro %}
 
 {% macro postgres__json_extract(from_table, json_column, json_path_list, normalized_json_path) -%}
-    jsonb_extract_path({{ from_table }}.{{ json_column }}, {{ format_json_path(json_path_list) }})
+    {%- if json_column|string() == "_airbyte_nested_data" %}
```

Comment:
This is tied to however we name the alias in 
- Postgres macro: https://github.com/airbytehq/airbyte/blob/5f4d263fe9ab34e3ac5741f33ac76b3335bea972/airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/array.sql#L28
- redshift macro: https://github.com/airbytehq/airbyte/blob/5f4d263fe9ab34e3ac5741f33ac76b3335bea972/airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/array.sql#L62
- etc

it would be nicer to find a way to pass the appropriate alias name in the `from_table` when the json_column is `_airbyte_nested_data` instead of making a special case in this macro coupled with the implementation of another macro